Orange/widgets/evaluate/owrocanalysis.py

```diff
@@ -336,6 +336,7 @@ def __init__(self):
         self._plot_curves = {}
         self._rocch = None
         self._perf_line = None
+        self._tooltip_cache = None
```

Comment:
I do not understand why the tooltip cache is necessary. Can you add a comment explaining the reasoning for its use?

Reply:
But it already checks all points in ` pts = sp.pointsAt(act_pos)`

Reply:
Right, but it also does it again later (although it is vectorized): `idx_closest = numpy.argmin(numpy.linalg.norm(diff, axis=1))`.

Considering your point, it does seem that the performance gain from tooltip cache might not be as big as I thought. Should I remove it and make the code clearer that way?